gymnasium/wrappers/time_aware_observation.py

```diff
@@ -27,6 +28,10 @@ def __init__(self, env: gym.Env):
         Args:
             env: The environment to apply the wrapper
         """
+        deprecation(
+            "`TimeAwareObservation` is marked as deprecated and will be removed in the near future."
+            "Use TimeAwareObservationV0 instead which allows support for composite action space and Jax."
+        )
```

Comment:
Is this needed here or should I remove it for now?

Reply:
I have changed how I think the development should work going forward https://github.com/Farama-Foundation/Gymnasium/pull/159

Reply:
https://github.com/Farama-Foundation/Gymnasium/pull/165/commits/a6c22fa77aecf49af3829e2c591c5abb7c41e8cb :+1: